### PR TITLE
CCOM-561 - Includes ESLint

### PR DESCRIPTION
**Pull request by alizeleal**
https://jirasoftware.catho.com.br/browse/CCOM-561

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,13 +29,35 @@
...
+    "prettier": "^1.15.3"
   },
   "scripts": {
     "dev": "next",
     "build": "next build",
     "start": "next start",
     "analyzer": "ANALYZE=true next",
-    "collect": "git checkout yarn.lock && rm -rf node_modules && yarn && yarn analyzer"
+    "collect": "git checkout yarn.lock && rm -rf node_modules && yarn && yarn analyzer",
+    "precommit": "lint-staged",
```
Comment (MarcosViniciusPC) [inline package.json:49]: The prepush was missing, I did tests and the lint problems are not blocking the push

**Review by MarcosViniciusPC — CHANGES_REQUESTED** [review]
[] Code review

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      }
+    }
+  ],
+  "globals": {
+    "window": true,
+    "document": true
+  }
+}
+
+/* eslint-disable react/no-unescaped-entities */
```
Comment (MarcosViniciusPC) [inline .eslintrc.json]:
```suggestion
```

**Review by MarcosViniciusPC — APPROVED** [review]
[x] Code review
[x] Commit and push

File: package.json (hunk 1 above)
Comment (rapahaeru) [reply package.json:49]: @alizeleal did you take this suggestion?

**Review by rapahaeru — APPROVED** [review]

**Pull request merged**